vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -816,7 +815,7 @@ private void addWorkflowVersion(HttpServerExchange exchange, AddWorkflowVersionR
                 var matchingWorkflow =
                     dsl.select(WORKFLOW.ID)
                         .from(WORKFLOW)
-                        .where(WORKFLOW.NAME.eq(name))
+                        .where(WORKFLOW.NAME.eq(param("workflowName", name)))
```

Comment:
I'm not quite following what the role of the first String argument is here - why is WORKFLOW.NAME associated with "name" at 757 and "workflowName" here?